Makefile

```diff
@@ -696,13 +699,13 @@ konflux-validate-catalog-template-bundle: yq operator-sdk
 		ENGINE=$(ENGINE)
 
 .PHONY: konflux-validate-catalog
-konflux-validate-catalog: opm ## validate the current catalog file
+konflux-validate-catalog: opm sync-git-submodules ## validate the current catalog file
```

Comment:
`sync-git-submodules` should be the first in any target list, I think, right?

Reply:
I did not consider that the order mattered but I think you make a good point here